### PR TITLE
rtl-sdr: update to 2.0.2 

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**

- Release notes for 2.0.1 can be found [here](https://osmocom.org/news/227).
- No release notes for 2.0.2 found.
- Resolves https://github.com/getsolus/packages/issues/2909

**Test Plan**

- Rebuilt soapyrtlsdr against this.

**Checklist**

- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/r/rtl-sdr/package.yml [hunk 1]
```diff
@@ -18,5 +18,3 @@ build      : |
 install    : |
     %make_install
     install -D -m00644 rtl-sdr.rules $installdir/%libdir%/udev/rules.d/10-rtl-sdr.rules
-
-    rm $installdir/usr/lib/*.a
```
Comment (algent-al) [inline packages/r/rtl-sdr/package.yml:22]: What was the reason to remove this line?

**Review by algent-al — CHANGES_REQUESTED** [review]

File: packages/r/rtl-sdr/package.yml (hunk 1 above)
Comment (androidnisse) [reply packages/r/rtl-sdr/package.yml:22]: It does not exist in the updated version:

+ rm '/home/build/YPKG/root/rtl-sdr/install/usr/lib/*.a'
rm: cannot remove '/home/build/YPKG/root/rtl-sdr/install/usr/lib/*.a': No such file or directory

Comment (algent-al) [reply packages/r/rtl-sdr/package.yml:22]: Ok, I understand what happened. Static lib was added actually to `/usr/lib64/librtlsdr.a`
Use this instead `rm  $installdir/%libdir%/*.a`

**Review by algent-al — CHANGES_REQUESTED** [review]

**Review by algent-al — APPROVED** [review]
LGTM, thanks!

**Pull request merged**